src/main/webapp/js/datahandlers/collegequerybuilder.js

```diff
@@ -38,19 +39,19 @@ const ACCEPTANCE_RATE = 'latest.admissions.admission_rate.overall';
 const ACT_SCORE = 'latest.admissions.act_scores.midpoint.cumulative';
 const ACCEPTANCE_RANGE = '__range=0..0.6';
 const ACT_RANGE = '__range=25..36';
-const ANNUAL_COST = 
-    'latest.cost.avg_net_price.private';
-const FIRST_NET_COST = 
-    'latest.cost.net_price.private.by_income_level.0-30000';
-const SECOND_NET_COST = 
-    'latest.cost.net_price.private.by_income_level.30001-48000';
-const THIRD_NET_COST = 
-    'latest.cost.net_price.private.by_income_level.48001-75000';
-const FOURTH_NET_COST = 
-    'latest.cost.net_price.private.by_income_level.75001-110000';
-const FIFTH_NET_COST = 
-    'latest.cost.net_price.private.by_income_level.110001-plus';
-const MEDIAN_DEBT = 'latest.aid.median_debt.completers.overall';
+const ANNUAL_COST =
+    '2016.cost.program_reporter.program_1.cip_6_digit.annualized';
+const FIRST_NET_COST = '2016.cost.net_price.private.by_income_level.0-30000';
+const SECOND_NET_COST =
+    '2016.cost.net_price.private.by_income_level.30001-48000';
+const THIRD_NET_COST =
+    '2016.cost.net_price.private.by_income_level.48001-75000';
+const FOURTH_NET_COST =
+    '2016.cost.net_price.private.by_income_level.75001-110000';
+const FIFTH_NET_COST =
+    '2016.cost.net_price.private.by_income_level.110001-plus';
+const MEDIAN_DEBT = '2016.aid.median_debt.completers.overall';
```

Comment:
why is this being changed to old data?

Reply:
Oh I didn't realized the change when rebasing to devel.
Just fixed it